submissions/oluwapelumiAlabi/productPage/index.html

```diff
@@ -19,10 +19,10 @@
         <h1 class="brand">UK IN NIGERIA STORE</h1>
         <nav>
           <ul>
-            <li><a href="#">Home</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Reviews</a></li>
-            <li><a href="#">Contact</a></li>
+            <li><a href="/">Home</a></li>
```

Comment:
/ doesnt specify a path name. You should have something like ../product-page-route/index.html